File: Java/백준/Bronze/2577. 숫자의 개수/숫자의 개수.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        int total = a * b * c;
        int[] numCountArr = new int[10];
        while (total > 0) {
            numCountArr[total % 10] += 1;
            total /= 10;
        }

        for (int numCount : numCountArr) {
            System.out.println(numCount);
        }
    }
}

File: Swift/백준/Bronze/2577. 숫자의 개수/숫자의 개수.swift
import Foundation

let a = Int(readLine()!)!
let b = Int(readLine()!)!
let c = Int(readLine()!)!

var total = a * b * c
var numCountArr = Array(repeating: 0, count: 10)

while total > 0 {
    numCountArr[total % 10] += 1
    total /= 10
}

for numCount in numCountArr {
    print(numCount)
}